app/views/publish/providers/accredited_providers/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :page_title, title_with_error_prefix("Enter a provider name, UKPRN or postcode", nil) %>
```

Comment:
![image](https://user-images.githubusercontent.com/470137/236423271-4dcec8bc-c7ec-455f-b292-d23631c879f1.png)


What is `nil`?

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/3547/commits/7caf382585be08f9efba307a6bf409f8550025d2